packages/tcore-console/scripts/icon.ts

```diff
@@ -32,7 +32,7 @@ const template = [
     .filter((x) => !x.startsWith(".")) // can't start with a dot
     .map(
       (x) =>
-        `  "${x.replace(".svg", "")}": require("../../../assets/icons/${x}"),`,
+        `  "${x.replace(".svg", "")}": lazy(() => ("../../../assets/icons/${x}")),`,
```

Comment:
I have not tested this script, but it should work to generate the `Icon.types.ts` file automatically from the `*.svg` assets

Reply:
Keep in mind that removing `lazy()` to use only `import()` doesn't work correctly in this case